pkg/cmd/server/start.go

```diff
@@ -253,6 +254,13 @@ func (o *WardleServerOptions) Config() (*apiserver.Config, error) {
 	serverConfig.FeatureGate = featuregate.DefaultComponentGlobalsRegistry.FeatureGateFor(featuregate.DefaultKubeComponent)
 	serverConfig.EffectiveVersion = featuregate.DefaultComponentGlobalsRegistry.EffectiveVersionFor(apiserver.WardleComponentName)
 
+	serverConfig.BuildHandlerChainFunc = func(apiHandler http.Handler, c *genericapiserver.Config) http.Handler {
```

Comment:
nice find, as you see below (line 301) we used `server.GenericAPIServer.Handler.FullHandlerChain` to alter the chain - yours might be a better solution

Reply:
I did not want to touch, I wasn't sure what is better

Reply:
definitely, I think it's good this way